packages/api/api.go

```diff
@@ -281,6 +293,35 @@ func NewServer(ctx context.Context, drv *sql.Driver, index bool, network string)
 	r.Handle("/playground", playground.Handler("GraphQL playground", "/query"))
 	r.Handle("/query", srv)
 
+	// run game server loop in the background
+	go gameState.Start(ctx)
+
+	r.HandleFunc("/game/ws", func(w http.ResponseWriter, r *http.Request) {
+		_, log := base.LogFor(ctx)
```

Comment:
You wanna use the request context here, rather than the outer scopes
```suggestion
		_, log := base.LogFor(r.Context())
```